### ISSUE TITLE
Nested lists styling broken

### ISSUE THREAD

**Issue by dGilli** [verbb/vizy#37]
**Description**
Nested `<ul>` not styled properly

**Additional info**
- Plugin version: 1.0.2
- Craft version: 3.6.15
- Multi-site: Yes

**Additional context**
<img width="564" alt="Screenshot 2021-05-31 at 19 30 56" src="https://user-images.githubusercontent.com/14869578/120226111-d612fd00-c246-11eb-8896-f7c5b8678a48.png">

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
